core/utils: Introduce shortString()

nhoriguchi opened the pull request:
This pull request suggests to introduce an upper limit of payload string in replica's log.

Resolves #111

ynamiki commented on core/utils.go:
```diff
@@ -75,6 +75,13 @@ func isPrimary(view uint64, id uint32, n uint32) bool {
 	return uint64(id) == view%uint64(n)
 }
 
+func getTrimmedMsg(msg string) string {
```
How about naming this function just "trim" or "trimMessage"? I imagine retrieving a value from a member variable of a object from the word "get".

nhoriguchi replied:
Thanks, I'll do it.

sergefdrv replied:
One more option for naming: `shortString`.

sergefdrv commented on core/utils.go:
```diff
@@ -75,6 +75,13 @@ func isPrimary(view uint64, id uint32, n uint32) bool {
 }
 
+func getTrimmedMsg(msg string) string {
+	if len(msg) > logMaxPayloadWidth {
```
There is a bit of dissonance between the function name and the name of this constant. The constant name relates to the payload, whereas the function name doesn't seem to reflect that. Maybe rename the constant more generically, or pass its value as a parameter.

nhoriguchi commented:
Thank you for comments, I've updated the branch by force-pushing the branch. The summary of updates is:

- rebased onto the latest master branch
- renamed the function and the constant
- introduced another parameter `max` to `shortString()`.

The diff between v1 and v2 is [here](https://github.com/Naoya-Horiguchi/minbft/compare/3971cbb...c483293), which might not be easy to read due to the rebase.

Commit:
core/utils: Introduce shortString()

Currently messageString() shows the whole message payload of REQUEST
and REPLY message, which makes us hard to read log if the massage conveys
large payload.  So this patch suggests to introduce an upper limit
of the payload string in the messages.

Signed-off-by: Naoya Horiguchi <n-horiguchi@ah.jp.nec.com>

nhoriguchi commented:
I force-pushed again to fix DCO failure. No functional change.
